perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -518,6 +592,178 @@ def SingleStreamThroughputBenchmark(storage_schema, host_to_connect=None):
                   host_to_connect=host_to_connect)
 
 
+def RunMultiThreadedWorkersWriteResults(worker, worker_args,
+                                        storage_schema, host_to_connect=None):
+  """Run a worker function in many threads, then gather and print the results.
+
+  Args:
+    worker: either WriteWorker or ReadWorker. The worker function to call.
+    worker_args: a tuple. The arguments to pass to the worker function. The
```

Comment:
I do pass in a tuple, so the documentation is correct here. According to the Python threading docs, a tuple is the correct type.